lib/presentation/widget/myspace/my_space_widget.dart

```diff
@@ -210,15 +208,13 @@ class _MySpaceWidgetState extends State<MySpaceWidget> {
       BuildContext context, dartz.Either<Failure, Success> uploadFileState) {
     uploadFileState.fold(
         (failure) => appToast.showToast(
-            AppLocalizations.of(context).stringOf('upload_failure_text')),
+            AppLocalizations.of(context).upload_failure_text),
         (success) => {
-              if (success is UploadFileSuccess)
-                {
-                  appToast.showToast(AppLocalizations.of(context)
-                      .stringOf('upload_success_text')),
-                  mySpaceViewModel.cleanUploadViewState()
-                }
-            });
+          if (success is UploadFileSuccess) {
+            appToast.showToast(AppLocalizations.of(context).upload_success_text),
```

Comment:
sr, this was a template static text i did before. Can u change this to dynamic text as design.
smthg like this:
<img width="430" alt="Screen Shot 2020-11-10 at 22 47 54" src="https://user-images.githubusercontent.com/16035728/98696953-f5c7fe00-23a6-11eb-90f6-59aab19f23d9.png">

Reply:
No, it is not scoped in this ticket :). We have the mechanism. We will do it later.